include/ginkgo/core/matrix/csr.hpp

```diff
@@ -971,6 +971,23 @@ class Csr : public EnableLinOp<Csr<ValueType, IndexType>>,
             gko::detail::array_const_cast(std::move(row_ptrs)), strategy});
     }
 
+    /*
+     * This is version of create_const with a default strategy.
+     */
+    static std::unique_ptr<const Csr> create_const(
+        std::shared_ptr<const Executor> exec, const dim<2>& size,
+        gko::detail::ConstArrayView<ValueType>&& values,
+        gko::detail::ConstArrayView<IndexType>&& col_idxs,
+        gko::detail::ConstArrayView<IndexType>&& row_ptrs)
+    {
+        // cast const-ness away, but return a const object afterwards,
+        // so we can ensure that no modifications take place.
+        return std::unique_ptr<const Csr>(new Csr{
+            exec, size, gko::detail::array_const_cast(std::move(values)),
+            gko::detail::array_const_cast(std::move(col_idxs)),
+            gko::detail::array_const_cast(std::move(row_ptrs))});
+    }
```

Comment:
same here, can be represented as a call to create_const with strategy parameter make_default_strategy(exec). AFAIK nnz doesn't need to be known to initialize the strategy.